docs/features/collision/collision.md

```diff
@@ -5,11 +5,137 @@ title: Collision
 description: 2D grid-based collision system.
 ---
 
-:::warning[Not up to date]
-This page **is not** up to date for MonoGame.Extended `@mgeversion@`.  If you would like to contribute to updating this document, please [create a new PR](https://github.com/craftworkgames/craftworkgames.github.io/pulls)
+:::tip[Up to date]
+This page is **up to date** for MonoGame.Extended `@mgeversion@`.  If you find outdated information, [please open an issue](https://github.com/craftworkgames/craftworkgames.github.io/issues).
 :::
 
-## Usage
+## Collision basics
```

Comment:
Should there be a basics section and a detailed explanation section?  If possible, documents shouldn't be split between tl;dr and full.

Reply:
I was trying to provide a high level overview since it's actually a bit complex to implement collision until you've done it once or twice.

Do you want me to remove the Section, or rename it?

Reply:
Let's circle back around on this at a later date.  The information on collisions can be split int separate documents if needed to provide a focused page covering a specific idea such as the algorithms and layers, similar to how the Texture Handling documents are broken into three documents that build on each other to go from TextureRegion to TextureAtlas to Sprite. 

I think for now the current organization of the document should be good and we can get feedback from users and see if any questions pop up.